app/design/frontend/Magento/blank/web/css/source/_navigation.less

```diff
@@ -135,8 +135,15 @@
         .switcher-dropdown {
             .lib-list-reset-styles();
             padding: @indent__s 0;
+            display: none;
         }
-
+        .switcher-options{
```

Comment:
Hi @cedarvinda. Thanks for collaboration. Please fix code indents after class declaration.